### PR TITLE
add mutual TLS info and a couple of minor edits

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview currently available here: https://nadine.preview.docs.r3.com/en/platform/corda/5.0-beta/operating/mutual-tls.html

### CODE REVIEW

File: content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mutual-tls.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+{{< note >}}
+* Mutual TLS is set per cluster. It must apply to all groups that the cluster hosts and all clusters that host those groups. You can not onboard a member unless the TLS type of the MGM cluster is aligned with the TLS type of the member cluster.
+* Changing the TLS type after a member or an MGM was onboarded, makes any TLS connection with that member unusable.
+* A virtual node can only be configured with a single TLS certificate that will be used as both a client and a server certificate.
+* A gateway accepts a TLS connection that uses a certificate associated with any member of any application network (partially) hosted in that cluster.
+{{< /note >}}
+
+## Modify the Cluster Configurations
+
+To configure a cluster to use mutual TLS, you must set the `sslConfig.tlsType` flag in the `corda.p2p.gateway` configuration to `MUTUAL` for the follwoing:
```
Comment (colin-hillis) [inline content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mutual-tls.md:22]:
```suggestion
To configure a cluster to use mutual TLS, you must set the `sslConfig.tlsType` flag in the `corda.p2p.gateway` configuration to `MUTUAL` for the following:
```

File: content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mutual-tls.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+* The MGM cluster before registering the MGM
+* All member clusters before uploading the CPI
+
+### Enable Mutual TLS Using Bash
+
+If using Bash, to configure the gateway SSL configuration to enable mutual TLS:
```
Comment (colin-hillis) [inline content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mutual-tls.md:28]:
```suggestion
If using Bash, perform the following steps to enable mutual TLS by configuring the gateway SSL:
```

File: content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mutual-tls.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+   ```
+   This command overwrites the revocation check setting. If you chose to disable revocation checks, use the following command instead:
+
+   ```shell
+   curl -k -u admin:admin -X PUT -d '{"section":"corda.p2p.gateway", "version":"'$CONFIG_VERSION'", "config":"{ \"sslConfig\": { \"tlsType\": \"MUTUAL\" , \"revocationCheck\": {\"mode\" : \"OFF\"} } }", "schemaVersion": {"major": 1, "minor": 0}}' $API_URL"/config"
+   ```
+
+### Enable Mutual TLS Using PowerShell
+
+If using PowerShell, to configure the gateway SSL configuration to enable mutual TLS, run the following:
```
Comment (colin-hillis) [inline content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mutual-tls.md:55]:
```suggestion
If using PowerShell, perform the following steps to enable mutual TLS by configuring the gateway SSL:
```

File: content/en/platform/corda/5.0-beta/operating/mutual-tls.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-mutual-tls
+    parent: corda-5-beta-operate
+    weight: 1000
+section_menu: corda-5-beta
+---
+Corda 5 uses TLS to secure a connection between two clusters. While establishing a TLS connection between the gateways of two clusters, the server gateway sends its certificate to the client gateway. The client gateway verifies the server certificate using its trust root certificate. In mutual TLS, in addition to that, the server gateway also requests the client gateway to send a client certificate and verify that it is using its trust root certificate.
+
+As the gateway manages the TLS connections for an entire cluster, the TLS mode (mutual or one-way) is defined in the gateway configuration and applies to the entire cluster. As a result, any group hosted in a mutual TLS cluster has to be a mutual TLS group, and all its members must be hosted on a mutual TLS cluster.
```
Comment (colin-hillis) [inline content/en/platform/corda/5.0-beta/operating/mutual-tls.md:13]: has to be a mutual TLS group
>>>>
must be a mutual TLS group

File: content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mutual-tls.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    identifier: corda-5-beta-mgm-onboarding-tls
+    parent: corda-5-beta-tutorials-deploy-dynamic
+    weight: 4000
+section_menu: corda-5-beta
+---
+As described in the [mutual TLS introduction](../../mutual-tls.md), you can configure the clusters in a dynamic network to connect using mutual TLS. This requires additional steps when onboarding an MGM or member into the dynamic network.
+
+{{< note >}}
+* Mutual TLS is set per cluster. It must apply to all groups that the cluster hosts and all clusters that host those groups. You can not onboard a member unless the TLS type of the MGM cluster is aligned with the TLS type of the member cluster.
+* Changing the TLS type after a member or an MGM was onboarded, makes any TLS connection with that member unusable.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mutual-tls.md:15]:
```suggestion
* Changing the TLS type after a member or an MGM was onboarded makes any TLS connection with that member unusable.
```

**Review by paulmoloneyr3 — DISMISSED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yift-r3 — APPROVED** [review]

**Review by colin-hillis — APPROVED** [review]
LGTM - Thank you Nadine

**Pull request merged**